snapshot/lib/snapshot/test_command_generator_base.rb

```diff
@@ -17,12 +17,13 @@ def project_path_array
         UI.user_error!("No project/workspace found")
       end
 
-      def options
+      def options(language, locale)
         config = Snapshot.config
         options = []
         options += project_path_array
         options << "-sdk '#{config[:sdk]}'" if config[:sdk]
         options << "-derivedDataPath '#{derived_data_path}'"
+        options << "-resultBundlePath '#{result_bundle_path(language, locale)}'" if config[:result_bundle]
```

Comment:
nit: I'm not a fan of method invocation during string interpolation.
it's not a big at all, just a note, and feel free to ignore me if you disagree 👍

Reply:
Ehhhh, I also feel the same way 😬 It _was_ already a method so it _looked_ okay but then I added parameters and then if got ugly so I 💯 agree with you. I can rework 😉